examples/unconditional_image_generation/train_unconditional.py

```diff
@@ -171,13 +171,26 @@ def parse_args():
     parser.add_argument(
         "--hub_private_repo", action="store_true", help="Whether or not to create a private repository."
     )
+    parser.add_argument(
+        "--logger",
+        type=str,
+        default="tensorboard",
+        choices=["tensorboard", "wandb"],
+        nargs="+",
```

Comment:
As said before we would like to only run one logger and not multiple loggers at once here